cloud-inquisitor/aws_s3_resource.go

```diff
@@ -253,3 +248,13 @@ func (s *AWSS3Storage) TakeAction(action Action) error {
 func (s3 *AWSS3Storage) GetType() Service {
 	return SERVICE_AWS_S3
 }
+
+func (s3 *AWSS3Storage) GetMetadata() map[string]interface{} {
+	return map[string]interface{}{
+		"account":      s3.AccountID,
+		"region":       s3.Region,
+		"bucketName":   s3.BucketName,
+		"currentState": s3.State,
```

Comment:
Is it possible to make `State` a string instead of an integer? It would be more human readable in the code and would change this from `map[string]interface{}` to `map[string]string`

Reply:
@rnikoopour will keep metadata as `map[string]interface{}` to allow nesting of other maps/structs and all data types.